File: app/src/main/java/com/akvelon/myfirstapp/lesson7/data/repository/weather/MvpWeatherRepository.java
package com.akvelon.myfirstapp.lesson7.data.repository.weather;

import androidx.annotation.NonNull;

import com.akvelon.myfirstapp.lesson4.rxweather.data.models.DayForecastResponse;

import io.reactivex.Single;

public interface MvpWeatherRepository {

    Single<DayForecastResponse> getDayForecast(@NonNull String city);

}

File: app/src/main/java/com/akvelon/myfirstapp/lesson9/data/repository/weather/MvpWeatherRepository.kt
package com.akvelon.myfirstapp.lesson9.data.repository.weather

import com.akvelon.myfirstapp.lesson9.data.models.DayForecastResponse
import io.reactivex.Single

interface MvpWeatherRepository {

    fun getDayForecast(city: String): Single<DayForecastResponse>

}
